spring-cloud-starter-aws-parameter-store-config/src/test/java/org/springframework/cloud/aws/autoconfigure/paramstore/AwsParamStoreBootstrapConfigurationTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.autoconfigure.paramstore;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import com.amazonaws.AmazonWebServiceClient;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClient;
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.springframework.cloud.aws.paramstore.AwsParamStoreProperties;
+
+/**
+ * Unit test for {@link AwsParamStoreBootstrapConfiguration}.
+ * @author Matej Nedic
+ */
+public class AwsParamStoreBootstrapConfigurationTest {
+
+	AwsParamStoreBootstrapConfiguration bootstrapConfig = new AwsParamStoreBootstrapConfiguration();
+
+	@Test
+	public void setRegion() throws NoSuchMethodException, InvocationTargetException,
+			IllegalAccessException {
+		String region = "us-east-2";
+		AwsParamStoreProperties awsParamStoreProperties = new AwsParamStorePropertiesBuilder()
+				.withRegion(region).build();
+
+		Method SSMClientMethod = AwsParamStoreBootstrapConfiguration.class
```

Comment:
May use `ReflectionUtils` from spring framework to do the reflection job

Reply:
Done.